src/Pheetup/CoreBundle/Controller/CrudController.php

```diff
@@ -114,9 +115,11 @@ public function viewAction($id)
         $repo = $this->getRepository();
         $view = [];
         $item = $repo->find($id);
-        $view["item"] = (array)$item;;
-        if (!$item->getGroup() instanceof Events) {
-            unset($view["item"][array_keys($view["item"])[6]]);
+        $view["item"] = (array)$item;
+        if (method_exists($item, 'getGroup')) {
+            if ($item->getGroup() instanceof Group) {
+                unset($view["item"][array_keys($view["item"])[6]]);

```

Comment:
Çok yanlış birşey yapmışsın. Sabit view kısmını değiştirmeni öneriyorum. @behramcelen 'in yapmış olduğu [EPDisplayBundle](https://github.com/EnterprisePHP/EPDisplayBundle) bu iş için şimdilik yeterli. Onu entegre edip buradaki silme olayını kaldırsan daha iyi olur gibi.